component-studio/component-runtime-di/src/main/java/org/talend/sdk/component/runtime/di/record/DiRowStructVisitor.java

```diff
@@ -397,14 +440,31 @@ private Entry toEntry(final String name, final Schema.Type type) {
     }
```

Comment:
toEntry(final String name, final Schema.Type type)
Is it used?

Reply:
good catch, I missed it 👍🏻